syntax = "proto2";

message CloseSwap {
    required bytes swapID = 1;
    required bytes secret = 2;
    }

